### PR TITLE
Meetup groups

### PR DESCRIPTION
**Pull request by Guitarbum722**
Add methods to return a slice of Meetup groups by specifying the organizer ID, the group ID, or the group's URL name.

### CODE REVIEW

**Comment by Guitarbum722** [comment]
@briandowns if you could review at your convenience, that would be awesome!

File: groups.go [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+const (
+	groupsEndpoint = "/2/groups"
+)
+
+// GroupByID returns the data for a single meetup group using the specified groupID
+// The response contains an array of results, even if there is only one because the request can
+// consist of comma separated values as the group_id parameter.
+func (c *Client) GroupByID(groupIDs []int) (*models.Groups, error) {
+	var groups models.Groups
+	var convIDs []string
```
Comment (briandowns) [inline groups.go]: You can save allocations by creating this slice like this: `convIDs := make([]string, 0, len(groupIDs))`.

File: groups.go [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+const (
+	groupsEndpoint = "/2/groups"
+)
+
+// GroupByID returns the data for a single meetup group using the specified groupID
+// The response contains an array of results, even if there is only one because the request can
+// consist of comma separated values as the group_id parameter.
+func (c *Client) GroupByID(groupIDs []int) (*models.Groups, error) {
+	var groups models.Groups
```
Comment (briandowns) [inline groups.go]: I'd likely decalre this more towards line 32 where it's closer to being used.

File: groups.go [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+	}
+
+	return &groups, nil
+}
+
+// GroupByURLName returns the data for a single meetup group using the specified urlName
+// The response contains an array of results, even if there is only one because the request can
+// consist of comma separated values as the group_id parameter.
+func (c *Client) GroupByURLName(urlNames []string) (*models.Groups, error) {
+	var groups models.Groups
```
Comment (briandowns) [inline groups.go]: Same remark as above. Declare closer to use.

**Comment by Guitarbum722** [comment]
Thanks @briandowns !

**Pull request merged**